src/Majora/Framework/Loader/Bridge/Doctrine/DoctrineLoaderTrait.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Majora\Framework\Loader\Bridge\Doctrine;
+
+use Majora\Framework\Loader\LoaderTrait;
+
+/**
+ * Trait to use into Doctrine loaders to get a simple implementation of LoaderInterface
+ */
+trait DoctrineLoaderTrait
+{
+    use LoaderTrait;
+
+    /**
+     * checks if loader is initialized.
+     *
+     * @throws RuntimeException if not configured
```

Comment:
`@throws \RuntimeException if not configured`

Il manque l'antislash, pour les IDE dignes de ce nom qui colorent même les docblock :trollface: